Library/Homebrew/dev-cmd/man.rb

```diff
@@ -189,10 +189,12 @@ def target_path_to_format(target)
   def generate_cmd_manpages(glob)
     cmd_paths = Pathname.glob(glob).sort
     man_page_lines = []
+    man_args = Homebrew.args
     cmd_paths.each do |cmd_path|
       begin
         cmd_parser = Homebrew.send(cmd_arg_parser(cmd_path))
         man_page_lines << cmd_manpage_lines(cmd_parser).join
+        Homebrew.args = man_args
```

Comment:
Does this need reset for every invocation of the loop?

Reply:
No.

Reply:
I thought within the loop there could be a method which might need `man_args`, but that doesn't happen here. So I'll move this line out of loop.